package.json

```
@@ -25,6 +25,7 @@
     "clsx": "^2.1.1",
     "i18next": "^23.16.0",
```

Biome (2.3.13)

[error] The key i18next was already declared.

This is where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)